src/config.rs

```diff
@@ -272,8 +274,29 @@ fn parse_proxy_config() -> Result<ProxyConfig, Error> {
     construct_proxy_config(mesh_config_path, pc_env).map_err(Error::ProxyConfig)
 }
 
+const IPV6_DISABLED_LO: &str = "/proc/sys/net/ipv6/conf/lo/disable_ipv6";
```

Comment:
If we wanted to get really wild we could actually have `istio-cni` inform us of this at the app pod level via zds.